common.js

```diff
@@ -14,14 +14,14 @@ var vcwg = {
       ],
       publisher: "University of California, Irvine."
     },
-    "VC-EXTENSION-REGISTRY": {
-      title: "Verifiable Credentials Extension Registry",
-      href: "https://w3c-ccg.github.io/vc-extension-registry/",
+    "VC-SPECS": {
```

Comment:
```suggestion
    "VC-SPECS-DIR": {
```

Maybe a minor point, but just `VC-SPECS` is pretty confusing... 
this change if accepted will impact multiple lines in index.html.

Reply:
I agree with this comment, but this change will need to be made everywhere the reference is, and the spec title is what matters most, this is just a ref pointer.

Reply:
Why is "VC-SPECS" confusing as a reference marker, rather than "VC-SPECS-DIR"... what does the "DIR" add that is important to a reader?

Reply:
The spec title is more important than an internal document pointer.

The issue is people will probably assume the document has more standing than it does, but imo it's not really our problem, that is W3C problem.